core/domain/config_domain.py

```diff
@@ -334,10 +344,10 @@ def get_config_property(cls, name):
         Returns:
             instance. The instance of the specified configuration property.
         """
-        return cls._config_registry.get(name)
+        return cls._config_registry.get(name) # type: ignore[return-value]
 
     @classmethod
-    def get_config_property_schemas(cls):
+    def get_config_property_schemas(cls) -> Dict[str, Dict[str, Any]]:
```

Comment:
Explain why this returns `Any`

Reply:
This function returns schema_dict which is of type :
{
    property_name: {
                                    schema: instance.schema,
                                    description: instance.description,
                                    value: instance.value
                              }
}

so, here first string refers to property_name, second string refers to (schema, description, value) and Any refers to values of instance.schema(Dict[str, Any]), instance.description(str), instance.value(Any)

Reply:
again add a code comment

Reply:
done!